cypress/integration/markdown_widget_backspace_spec.js

```diff
@@ -0,0 +1,82 @@
+import '../utils/dismiss-local-backup';
+
+describe('Markdown widget', () => {
+
+  before(() => {
+    Cypress.config('defaultCommandTimeout', 4000);
+    cy.task('setupBackend', { backend: 'test' });
+    cy.loginAndNewPost();
+  });
+
+  beforeEach(() => {
+    cy.clearMarkdownEditorContent();
+  });
+
+  after(() => {
+    cy.task('teardownBackend', { backend: 'test' });
+  });
+
+  describe('pressing backspace', () => {
+    it('sets non-default block to default when empty', () => {
```

Comment:
These tests, as well as the others, execute multiple checks without fully resetting the environment. It isn't ideal as failures can cascade, but it saves a whole lot of time.